src/sources/journald.rs

```diff
@@ -209,6 +209,10 @@ pub struct JournaldConfig {
     #[configurable(metadata(docs::hidden))]
     #[serde(default)]
     log_namespace: Option<bool>,
+
+    /// Whether to emit the __CURSOR field
```

Comment:
Can you add a link to the documentation? Also, can you run `make generate-component-docs` to update the user-facing documentation? It'll automatically generate the changes from the code changes.

```suggestion
    /// Whether or not to emit the [__CURSOR field][cursor].
    ///
    /// [cursor]: https://www.freedesktop.org/software/systemd/man/latest/systemd.journal-fields.html#Address%20Fields
```

Reply:
Thanks, done.